env0/resource_team_organization_assignment.go

```diff
@@ -29,7 +29,7 @@ func resourceTeamOrganizationAssignment() *schema.Resource {
 				Type:             schema.TypeString,
 				Description:      "id of the assigned custom role. The following built-in roles can be passed as well: `User`, `Admin`",
 				Required:         true,
-				ValidateDiagFunc: ValidateNotEmptyString,
+				ValidateDiagFunc: NewRoleValidator([]string{"User", "Admin"}),
```

Comment:
I feel like `[]string{"User", "Admin"}` should be a constant instance that can be imported from `builtin_roles.go`
same for other scopes

Reply:
Sure.